common/services/person.js

```diff
@@ -32,75 +31,77 @@ const personService = {
       relationship,
       date,
     })
-  },
+  }
 
   create(data) {
-    return apiClient
-      .create('person', personService.format(data))
+    return this.apiClient
+      .create('person', this.constructor.format(data))
```

Comment:
Without the static, this can become `this.format` I think. Again, I wonder if this is some TS stuff sneaking in.